admin/src/graphql/createPendingCreation.js

```diff
@@ -3,7 +3,7 @@ import gql from 'graphql-tag'
 export const createPendingCreation = gql`
   mutation (
     $email: String!
-    $amount: Int!
+    $amount: Float!
```

Comment:
I do not like this at all. `amount` is definitely an integer. Did you try `parseInt(amount)` when setting the variables for the mutation?

Reply:
I though that it was something that we had to change!?
We want to create 1000,50 GDD or do we want to fix it to integers?

Reply:
Was the point "Use float instead of integer" What did we mean with this?